README.md

```diff
@@ -203,33 +197,36 @@ If you walk through the cells, you will see that we demonstrated how to do the f
 * Do some data manipulation with pandas
 * Use BeautifulSoup
 * Use Natural Language Understanding
-* Use Tone Analyzer
 * Use Visual Recognition
 * Save the enriched data in a CSV file in Object Storage
 
 ### Part II - Data Preparation
 
-In Part II, we used pandas to create multiple DataFrames from our main enriched DataFrame. After slicing and dicing and cleaning, these new DataFrames are ready for PixieDust to use.
+In Part II, we used pandas to create multiple DataFrames from our main enriched DataFrame.
 
 ### Part III - Analyze
 
-In Part III, we analyze the results by exploring and visualizing the metrics with PixieDust.
+In Part III, we analyze the results by exploring and visualizing the metrics.
 
-After all the prep work done earlier, you'll see that there is almost no code needed here (thanks to PixieDust). We just use one-liners like this:
+After all the prep work done earlier, you'll see that there is almost no code needed here. We just use one-liners like this:
 
 ```python
 display(<data-frame>)
 ```
 
 You should also notice that we used ```display(tones)``` in two different cells, but the result was two different charts. How can that happen? Well, we used cell metadata to tell PixieDust how to display the data. Notice the `Edit Metadata` button on each cell. If you don't see it, use the menu `View > Cell Toolbar > Edit Metadata` to make it visible. If you look at the metadata for the first two charts, you'll see how we got a bar chart and a pie chart.
 
+![emotional_engagement](doc/source/images/emotional_engagement.png)
+
+![sentimental_engagement](doc/source/images/sentimental_engagement.png)
+
 **PixieDust is interactive!** This is where we explore to find out what the enriched data will tell us.
```

Comment:
did you mean to leave this PixieDust section in?

Reply:
yeah it's not gone yet.  I guess the 2nd sentence would work on its own